### PR TITLE
Bug fixes: 32, 33, 37

### PR DESCRIPTION
**Pull request by nate-double-u**
- fixes https://github.com/cncf/hugo-netlify-starter/issues/32 adding linkTitle to the section nav, and sample use in the content
- fixes https://github.com/cncf/hugo-netlify-starter/issues/33 correct "Edit on GitHub" button
- fixes https://github.com/cncf/hugo-netlify-starter/issues/37 fix twitter button

### CODE REVIEW

**Comment by nate-double-u** [comment]
Deploy preview: https://deploy-preview-39--cncf-hugo-starter.netlify.app/

File: content/test-content/docs-section-1/_index.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Section 1 index"
+linkTitle: "Section 1 index and `linkTitle` Sample"
```
Comment (celestehorgan) [inline content/test-content/docs-section-1/_index.md:3]: The [Deploy preview](https://deploy-preview-39--cncf-hugo-starter.netlify.app/test-content/docs-section-1/) for this page doesn't seem to work correctly anymore?

File: layouts/partials/github-edit.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="col col-4">
     {{ $repoUrl := .Site.Params.repositoryUrl }}
     {{ $filePath := .File.Path }}
-    {{ $editUrl := print $repoUrl "/content/" $filePath }}
+    {{ $editUrl := print $repoUrl "/blob/main/content/" $filePath }}
```
Comment (celestehorgan) [inline layouts/partials/github-edit.html:4]: Awesome ✨

File: layouts/partials/social-buttons.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 
 {{ range . }}
 {{ $color := .color | default "white" }}
-<a class="btn text-light" style="background-color: {{ .color }}">
+<a class="btn text-light" style="background-color: {{ .color }}" href="{{ .url }}">
```
Comment (celestehorgan) [inline layouts/partials/social-buttons.html:4]: Awesome #2 ✨

**Review by celestehorgan — APPROVED** [review]
Thank you so much for contributing these!!

**Pull request merged**